### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by AlexxNica**
#### PR checklist

- [ ] Addresses an existing issue: #0000
- [ ] New feature, bugfix, or enhancement
- [ ] Includes tests
- [x] Documentation update

```diff
-paramter
+parameter
```

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tslint](https://github.com/palantir/tslint), @AlexxNica! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: _data/rules.json [hunk 1]
```diff
@@ -1470,7 +1470,7 @@
     "description": "Disallows duplicate variable declarations in the same block scope.",
     "descriptionDetails": "\nThis rule is only useful when using the `var` keyword -\nthe compiler will detect redeclarations of `let` and `const` variables.",
     "rationale": "\nA variable can be reassigned if necessary -\nthere's no good reason to have a duplicate variable declaration.",
-    "optionsDescription": "You can specify `\"check-parameters\"` to check for variables with the same name as a paramter.",
+    "optionsDescription": "You can specify `\"check-parameters\"` to check for variables with the same name as a parameter.",
```
Comment (adidahiya) [inline _data/rules.json]: ah, wait, you have to change this in the rule source itself, not the `_data` folder.

Comment (AlexxNica) [reply _data/rules.json]: Oops, my bad! 1 sec.

Comment (ajafff) [reply _data/rules.json]: This is already fixed on master

Comment (AlexxNica) [reply _data/rules.json]: @ajafff I just opened another PR fixing the same file under `src/`, idk if that was needed, so..

**Pull request closed without merging**